code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -107,19 +105,15 @@
 	..()
 	new /obj/item/clothing/suit/armor/vest(src)
 	new /obj/item/device/radio/headset/headset_sec(src)
-	new /obj/item/device/radio/headset/headset_sec/alt(src)
-	new /obj/item/weapon/reagent_containers/spray/pepper(src)
-	new /obj/item/device/assembly/flash/handheld(src)
-	new /obj/item/weapon/grenade/flashbang(src)
+	new /obj/item/device/radio/headset/headset_sec/alt(src) 
+	new /obj/item/clothing/head/helmet/sec(src)
 	new /obj/item/weapon/storage/belt/security/full(src)
 	new /obj/item/clothing/glasses/hud/security/sunglasses(src)
-	new /obj/item/clothing/head/helmet/sec(src)
 
 /obj/structure/closet/secure_closet/security/sec
 
 /obj/structure/closet/secure_closet/security/sec/New()
```

Comment:
Checked the map and `/obj/structure/closet/secure_closet/security/sec` and definitely used in a few places. I think having the New() override is okay incase people want to add in items in the future